x/ecocredit/server/core/create_project.go

```diff
@@ -40,8 +41,14 @@ func (k Keeper) CreateProject(ctx context.Context, req *core.MsgCreateProject) (
 		}
 	}
 
+	adminAddress, err := sdk.AccAddressFromBech32(req.Issuer)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
might be a micro-optimization but `AddressFromBech32` also gets called from `assertClassIssuer`. might be worth it to move this up, and change the sig of `assertClassIssuer` to take an AccAddress instead of string so it's only done once in the call

Reply:
https://github.com/regen-network/regen-ledger/pull/888/commits/040471bcc5c9199918153364ffa303bf348d25d8